### PR TITLE
Allow for quickpick commandline usage

### LINKED ISSUE #2773: (feature) Add an option to bring commandline back to old place
https://github.com/VSCodeVim/Vim/pull/2758 (released in v0.14.0) moves the commandline to the status bar to allow history navigation.

This forces me to activate the status bar.

It would be nice if there is an option provided to configure the place of the commandline.

### PR DESCRIPTION
**Pull request by xconverge**
fixes #2773

@jpoon I think that this doesn't add too much to maintain and was functionality that existed before

### CODE REVIEW

**Comment by jpoon** [comment]
I'm still of the position that we shouldn't do this considering there won't be feature parity between status bar commands vs input box commands. If we do this, we should re-open https://github.com/VSCodeVim/Vim/issues/779.

File: README.md [hunk 1]
```diff
@@ -597,6 +597,25 @@ Are you on a Mac? Did you go through our [mac-setup](#mac-setup) instructions?
...
+    "command": "vim.showQuickpickCmdLine",
+    "when": "editorTextFocus && vim.mode != 'Insert'"
+}
+```
+Or for Zen mode only:
+```
+{
+    "key": "shift+;",
+    "command": "vim.showQuickpickCmdLine",
+    "when": "inZenMode && vim.mode != 'Insert'"
```
Comment (jpoon) [inline README.md]: When and where is this context variable being set?

**Comment by jpoon** [comment]
Another possible feature disparity between the two: https://github.com/VSCodeVim/Vim/issues/745

File: README.md [hunk 2]
```diff
@@ -597,6 +597,25 @@ Are you on a Mac? Did you go through our [mac-setup](#mac-setup) instructions?
 
 Press `shift+<esc>` to close all of those boxes.
 
+### How can I use the commandline when in Zen mode or when the status bar is disabled?
+
+This extension registers a command pallete of visual studio code exists to show a quickpick version of the commandline. This can be remapped as follows in visual studio keybindings.json settings file.
```
Comment (xconverge) [inline README.md]: Fix grammar in this sentence

**Comment by jpoon** [comment]
So what's the verdict with this PR @xconverge?

**Comment by mindgitrwx** [comment]
Is it updated properly? I reinstalled vim plugin and reopen, but nothing have been happened. 

When I typed command shift + ;, It always brings me messageslike this: 'vim.showQuickPickCmdLine' not found

**Comment by jpoon** [comment]
It's not released yet. https://github.com/VSCodeVim/Vim/releases

**Pull request merged**